examples/grpc-kotlin/src/main/kotlin/example/armeria/grpc/kotlin/HelloServiceImpl.kt

```diff
@@ -42,13 +38,12 @@ class HelloServiceImpl : HelloServiceGrpc.HelloServiceImplBase() {
         // 3. Call a blocking API in the separate thread pool you manage.
         //
         // In this example, we chose the option 1:
-        ServiceRequestContext.current().blockingTaskExecutor().submit {
+        return runBlocking {
```

Comment:
This sleeps this thread. The original example was trying to emulate a background long running task running. Did we consider  using https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines/-dispatchers/-i-o.html ?

Reply:
I thought `runBlocking` submit the task to blocking thread pool. Thanks for pointing out. 👍
`withContext(Dispatchers.IO)` will work as expected.